green_cli/common.py

```diff
@@ -400,7 +401,9 @@ def _txlist_summary(txlist):
             tx['satoshi'] = {asset: -tx['satoshi'][asset] for asset in tx['satoshi']}
         for asset, amount in tx['satoshi'].items():
             balance[asset] += amount
-            lines.append(f"{tx['txhash']} {tx['created_at']} ({confs}) {amount:+} "\
+            ts = tx['created_at_ts']
+            created_at = datetime.fromtimestamp(ts / 1000000.0, tz=timezone.utc).replace(tzinfo=None)
```

Comment:
Do we want to format the date up to micro seconds? perhaps we want to do `ts // 1000000` so we round to seconds

Reply:
Since this is ust for summary display, that makes sense. Updated, thanks.